deployment/ansible/group_vars/all.example

```diff
@@ -17,7 +17,36 @@ docker_compose_version: 1.8.0
 # azavea.docker
 # 1.12.* to match AWS ECS
 docker_version: 1.12.*
+docker_py_version: 1.9.0
 docker_options: "--storage-opt dm.basesize=20G"
+docker_keyserver: hkp://p80.pool.sks-keyservers.net:80
 
 # azavea.terraform
 terraform_version: 0.9.1
+
+# application settings
+docker_compose_version: 1.8.0
+postgresql_database_name: "pfb"
```

Comment:
Nothing below this line should be necessary. All of these are old variables that are no longer in use.

Wouldn't hurt to keep the `docker_compose_version`, but should probably be moved up into the `azavea.docker` section of the file